### ISSUE TITLE
Radar Chart Bug

### ISSUE THREAD

**Issue by anipoli** [WilliamOpenshaw/Ethics_Calculator_Unity_Project#10]
The radar chart can be ore clear. Currently, if the user score shows a large percentage in one school and no percentages in the schools next it, it is impossible to see the "spike" or area of the radar chart. By doing this, the player can know what percentages of which school they scored in visually, and more accurately. We can do this by setting the location of 0% of each school slightly away from the central point.

Here are pictures of the problem:
<img width="1412" alt="Screenshot 2023-09-21 at 3 17 15 PM" src="https://github.com/WilliamOpenshaw/Ethics_Calculator_Unity_Project/assets/146810509/bb0b4a60-1b0d-4e5c-aff9-bd475bfeeed1">
<img width="1423" alt="Screenshot 2023-09-21 at 3 17 04 PM" src="https://github.com/WilliamOpenshaw/Ethics_Calculator_Unity_Project/assets/146810509/cc83d6ab-e838-4114-85d4-47fa97fcfa91">

**Comment by WilliamOpenshaw** [comment]
This is a big problem, and the solution you proposed is a good one.
I also wonder why there is not a line from Neoliberalism's 23 to Virtue Ethic's 5 on the Radar Chart.
I would think that Kantian being 0 would not influence the visibility of that line from 23 NL to 5 VE.
I will take a look at this and fix it, unless you want to do it yourself.
One thing I know is that for this radar chart, it needs 6 results in its code to display everything correctly, as 5 results in its code will just result in Virtue Ethics's result not being shown at all.
I thought that might be part of the problem here.
If I find a solution I will share it with you.

**Comment by WilliamOpenshaw** [comment]
I added a sixth value to the list of results on the radar chart because it doesn't work properly with just 5 values.
Not sure why, but it works now, so we can just do it this way for now until we run into another problem.
Maybe the original Radar Chart script ignores the 5th value, maybe to finish the pentagon shape it is trying to make it needs two values for the "top" point of the pentagon. We can play around with the values to find out.

![image](https://github.com/WilliamOpenshaw/Ethics_Calculator_Unity_Project/assets/16065395/4f75aa81-4c21-455c-87f5-1bdb5c3474fb)

radar chart now:

![image](https://github.com/WilliamOpenshaw/Ethics_Calculator_Unity_Project/assets/16065395/cf057126-976b-45ac-ac85-06104d393ba2)

Instead of making the lowest result of the test 0, I changed it so that the lowest result of the test would be 2.
The reason we do it this way is because when the test is done, some schools might have a negative value, which doesn't work for what we want. So we take all the results of the test for each school and add the lowest number to all of them as its absolute value.

To give an example, if someone gets the scores:
 (K , 20) (VE , 16) (NL , 5) (R , -5) (U , -13)
then we will take the lowest score: (U , -13), 
and take its absolute value: (13)
and add that to all the scores so that the scores become:
--> (K , 33) (VE , 29) (NL , 18) (R , 8) (U , 0)
This distorts the ratios between the scores a bit, but is better than negative values.

We found now that 0 is not great because it has problems showing up on the radar chart and breaks other values,
so instead of zero, the lowest value now is going to be 2, just in case 1 is not clear enough.

We can try different solution when we encounter another problem.

![image](https://github.com/WilliamOpenshaw/Ethics_Calculator_Unity_Project/assets/16065395/2cf21e96-aa13-4640-8567-8f0876532ff5)
